ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -30,4 +41,15 @@ spec:
           exit 1
         fi
 
+        PROJECT_URL="$(params.pyxis_url)/v1/projects/certification/id/$CERT_PROJECT_ID"
+        PROJECT=$(curl -k -X GET $PROJECT_URL -H  "accept: application/json" --cert $(params.pyxis_cert_path) --key $(params.pyxis_key_path))
```

Comment:
I discussed with Ales a bit about using a Python script to get the cert project and then extract all the needed fields, but that'll require refactoring operator-pipeline-images and will further delay this. I'd rather merge this first then refactor.